.github/workflows/e2e.yml

```diff
@@ -22,4 +22,7 @@ jobs:
           pip3 install --upgrade pip
           pip3 install setuptools
           pip3 install -r requirements.txt
+          until curl http://127.0.0.1:8081/healthCheck; do
+            sleep 1
+          done
```

Comment:
@kezhenxu94 Is this a good practice?

Reply:
> @kezhenxu94 Is this a good practice?

Ideally I’d put this check in the `run_e2e.py` so when running locally the healthiness is also checked.

Reply:
@kezhenxu94 that sounds reasonable. Let me push a follow up for it.